packages/esm-patient-attachments-app/webpack.config.js

```diff
@@ -3,15 +3,23 @@ const { resolve } = require('path');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
 const { StatsWriterPlugin } = require('webpack-stats-plugin');
-
 const { peerDependencies } = require('./package.json');
 
 const filename = 'openmrs-esm-patient-attachments-app.js';
 const cssLoader = {
   loader: 'css-loader',
   options: {
     modules: {
-      localIdentName: 'esm-patient-attachments_[name]__[local]___[hash:base64:5]',
+      mode: (resourcePath) => {
+        if (
+          /.*react-html5-camera-photo\/build\/css\/index.css/i.test(resourcePath) ||
+          /styles.css$/i.test(resourcePath)
+        ) {
+          return 'global';
+        }
+        return 'local';
+      },
+      localIdentName: 'esm-patient-chart-widgets__[name]__[local]___[hash:base64:5]',
```

Comment:
That is not a problem. You can still modify or roll your own webpack config.

Reply:
I posted that already in Slack, but since you obviously did not read it I re-post an example:

```js
const cfgCreator = require('openmrs/default-webpack-config');
module.exports = (env, argv) => {
  const config = cfgCreator(env, argv);
  // do your changes on config
  return config;
};
```

Reply:
Thanks @FlorianRappl for the hint. Updated :)